### PR TITLE
Resources landing page

### PR DESCRIPTION
**Pull request by clairehhelms**
resources page, resources sub-pages, and support page

### CODE REVIEW

File: components/NavigationBar.js [hunk 1]
```diff
@@ -186,13 +186,20 @@ export const NavigationBar = ({
             </Anchor>
           </Link>
         </ListItem>
-        <ListItem css={{ margin: "0 $100 0 0" }}>
+        <ListItem >
```
Comment (artmsilva) [inline components/NavigationBar.js]:
```suggestion
        <ListItem>
```

File: pages/index copy.js [hunk 2]
```diff
@@ -0,0 +1,255 @@
+import React from "react";
```
Comment (artmsilva) [inline pages/index copy.js]: is this intentional? "index copy.js"?

File: next.config.js [hunk 3]
```diff
@@ -57,4 +57,33 @@ module.exports = withBundleAnalyzer({
...
+  async redirects() {
+    return [
+      {
+        source: '/resources/architecture/stitches',
+        destination: '/resources/guides/stitches',
+        permanent: false,
+      },
+      {
+        source: '/resources/process/releases',
+        destination: '/supposrt/release-cycles',
```
Comment (artmsilva) [inline next.config.js]: Typo here

File: pages/index copy.js (hunk 2 above)
Comment (clairehhelms) [reply pages/index copy.js]: ooh no thought I deleted this, ty!

**Comment by clairehhelms** [comment]
going to need to undo a commit/delete the new architecture doc before merging so nothing breaks

File: .husky/pre-commit [hunk 4]
```diff
@@ -1,8 +1,5 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-echo "--> Installing libraries..."
-npm install
```
Comment (atomictangerine) [inline .husky/pre-commit]: why are we getting rid of this?

File: components/Thumbnail.js [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+  flex: "1",
+  [".wpds-dark &"]: { opacity: "0.9" },
+});
+
+const Grid = styled("section", {
+  width: "100%",
+  display: "grid",
+  variants: {
+    type: {
+      full: {
```
Comment (atomictangerine) [inline components/Thumbnail.js]: For the grid of three columns, maybe we could make it match the homepage? that way it stays consistent throughout the landing pages.

```const GridThreeColumns = styled("div", {
    display: "grid",
    gap: "$100",
    "@notSm": {
      gridTemplateColumns: "repeat(3,1fr)",
    },
    "@sm": {
      gridTemplateColumns: "1fr",
    },
  });```

File: docs/resources/guides/themes.mdx [hunk 6]
```diff
@@ -0,0 +1,205 @@
...
+
+## Stitches functions
+
+Stitches functions make creating and styling components easy. Below, we’ll dive into the functions most useful to a user of our UI Kit, but check out Stitches’ [other functions](https://stitches.dev/docs/api) if you’re curious.
+
+## Styled
+
+The `styled` function is used to simply create React components with styles. The arguments to the function are:
+
+1. either a HTML element or a react component
```
Comment (atomictangerine) [inline docs/resources/guides/themes.mdx]: nit: either `an` HTML

File: pages/resources/index.js [hunk 7]
```diff
@@ -5,133 +5,154 @@ import { Box, Icon, theme, styled } from "@washingtonpost/wpds-ui-kit";
...
   margin: "$100 0 $050",
-  padding: 0,
+  paddingTop: 0,
   border: 0,
   height: "1px",
   backgroundColor: "$subtle",
+  variants: { type: { Workshops: { display: "none" } } },
 });
 
 const CheveronForLink = styled(ChevronRight, {
```
Comment (atomictangerine) [inline pages/resources/index.js]: nit: typo on Chevron

File: pages/resources/index.js [hunk 8]
```diff
@@ -5,133 +5,154 @@ import { Box, Icon, theme, styled } from "@washingtonpost/wpds-ui-kit";
...
-                }}
+      <Header as="h1">Resources</Header>
+      <Description>
+        Learn more about our workflow and how to use our tools in our guides.
+        Watch our tutorials and workshops to discover the elegance and
+        accessibility of WPDS.
+      </Description>
+      <TableofContents headings={headings} css={{ margin: "$075 0" }} />
+      <>
+        {content.categories.map((cat) => {
```
Comment (atomictangerine) [inline pages/resources/index.js]: instead of cat could we possibly spell out `category`? It's not too hard to type out and it'll make things a bit clearer

File: pages/support/index.js [hunk 9]
```diff
@@ -0,0 +1,124 @@
+import React from "react";
+import { NextSeo } from "next-seo";
+import { getDocsListBySection, getNavigation } from "~/services";
+import { Box, Icon, theme, styled } from "@washingtonpost/wpds-ui-kit";
+import ChevronRight from "@washingtonpost/wpds-assets/asset/chevron-right";
+import { Header } from "~/components/Markdown/Components/headers";
+import Link from "~/components/Markdown/Components/link";
+import { P } from "~/components/Markdown/Styling";
+
+const Masonry = styled("section", {
```
Comment (atomictangerine) [inline pages/support/index.js:10]: I noticed that we're using grids all over the place (so good!) However, I'm thinking that maybe, since we use them all the time, we could pull them out to a utils or libs or something? 

We could do something like: 
```const GridThreeColumns = styled("div", {
    display: "grid",
    gap: "$100",
    "@notSm": {
      gridTemplateColumns: "repeat(3,1fr)",
    },
    "@sm": {
      gridTemplateColumns: "1fr",
    },
  });

  const GridTwoColumns = styled("div", {
    display: "grid",
    gap: "$200",
    "@notSm": {
      gridTemplateColumns: "repeat(2,1fr)",
    },
    "@sm": {
      gap: "$100",
      gridTemplateColumns: "1fr",
    },
  });``` 

or create a variant like you did earlier (with descriptive names like these). Thoughts?

**Review by atomictangerine — COMMENTED** [review]
Also, could we move the thumbnails folder up a level (aka keep the structure flat)? We might use those for something other than guides. Even if we don't, they're not so tied to guides that they need to be nested.

File: .husky/pre-commit (hunk 4 above)
Comment (clairehhelms) [reply .husky/pre-commit]: @artmsilva cut this when we were working together, but I can totally put it back if there's no reason to cut it

File: pages/resources/index.js [hunk 10]
```diff
@@ -156,12 +177,60 @@ export const getStaticProps = async ({ params }) => {
     return a.kicker.localeCompare(b.kicker);
   });
 
+  // compile page content based on kicker. Includes a name, the posts to be displayed, a description, and the size of the posts
+  // we have to hard-code What's New because it's too unique
+  let content = { categories: [] };
```
Comment (atomictangerine) [inline pages/resources/index.js]: This is a lot of stuff to add to the getStaticProps function. Maybe we could pull this out to its own function?

File: pages/resources/index.js [hunk 11]
```diff
@@ -156,12 +177,60 @@ export const getStaticProps = async ({ params }) => {
...
+      "Watch or read through our tutorials to understand key techniques and concepts.",
+    Workshops:
+      "Sharpen your design and development skills with our in-depth recorded workshops.",
+  };
+
+  for (let i = 0; i < collections.length; i++) {
+    name = collections[i].kicker;
+    if (name === "Guides") {
+      posts = [...collections[i].docs]
+        .sort((a, b) => a.data.guideRank - b.data.guideRank)
```
Comment (atomictangerine) [inline pages/resources/index.js]: What if the data doesn't have a guideRank?

File: pages/resources/index.js (hunk 10 above)
Comment (atomictangerine) [reply pages/resources/index.js]: This whole section is a little hard to follow. Is there a way that we could make it 1) more readable? and 2) add more comments, please?

File: pages/support/index.js (hunk 9 above)
Comment (clairehhelms) [reply pages/support/index.js:10]: love this!

File: pages/resources/index.js (hunk 11 above)
Comment (clairehhelms) [reply pages/resources/index.js]: No error is thrown and everything behaves correctly because Chrome handles invalid arguments by resolving to sort them alphanumerically! But I can still add error handling in case other browsers don't support this :-)

File: components/Thumbnail.js (hunk 5 above)
Comment (clairehhelms) [reply components/Thumbnail.js]: tried this but it doesn't look right with the new thumbnail design!

**Review by artmsilva — APPROVED** [review]

File: components/Thumbnail.js [hunk 12]
```diff
@@ -0,0 +1,74 @@
+import { styled } from "@washingtonpost/wpds-ui-kit";
+import { Header } from "~/components/Markdown/Components/headers";
+import { P } from "~/components/Markdown/Styling";
+import Image from "next/image";
+import { ThumbnailGrid } from "./Markdown/Components/ResourcesGrids";
+
+// image has 90% opacity when the page is on dark mode
+const StyledImage = styled(Image, {
+  flex: "1",
+  [".wpds-dark &"]: { opacity: "0.9" },
```
Comment (atomictangerine) [inline components/Thumbnail.js:13]: This is fancy stitch stuff! *writes down notes*

File: components/Thumbnail.js [hunk 13]
```diff
@@ -0,0 +1,74 @@
...
+
+const Content = styled("div", {
+  display: "flex",
+});
+
+export const Thumbnail = (props) => {
+  // switches based on the size of the post displayed – full for workshops and tutorials, mini for guides
+  // note the description is hidden for what's new and the publish date is hidden for tutorials on the resources landing page
+  switch (props.size) {
+    case "full":
```
Comment (atomictangerine) [inline components/Thumbnail.js]: can we make these cases constants that we import here and where we use them? That way we limit typos. Say I pass in 'minni' on accident, the styles won't work. However, if we use `THUMBNAIL` AND `THUMBNAIL_WIDE` (or something), they'll autocomplete and decrease user error.

Comment (atomictangerine) [reply components/Thumbnail.js]: you can export them from this thumbnail component.

File: components/Thumbnail.js [hunk 14]
```diff
@@ -0,0 +1,74 @@
...
+      return (
+        <ThumbnailGrid type="full">
+          <Content>
+            {props.imageTag && (
```
Comment (atomictangerine) [inline components/Thumbnail.js]: Just a suggestion (nothing wrong with how it's here) but it might be easier to see everything we're using from props if we do something like `const {imageTag, name, etc...} = props;` before the return?

File: pages/resources/[category]/index.js [hunk 15]
```diff
@@ -3,78 +3,68 @@ import { NextSeo } from "next-seo";
 import Header from "~/components/Typography/Headers";
 import CustomLink from "~/components/Typography/link";
 import { getAllPathsByCategory, getNavigation, getResources } from "~/services";
-import { P } from "~/components/Markdown/Styling";
 import Breadcrumbs from "~/components/Breadcrumbs";
+import { Thumbnail } from "~/components/Thumbnail";
+import { SubPageGrid } from "~/components/Markdown/Components/ResourcesGrids";
```
Comment (atomictangerine) [inline pages/resources/[category]/index.js]: Probs a nit but, do we have a `PageGrid`? I guess, I have trouble understand what the SubPageGrid really is based on the name. Is it possible to use this `SubPageGrid` somewhere else? Could we name it something more generic so that we don't feel restricted by the name (unless it can only be used as a sub page, that is)?

File: pages/resources/[category]/index.js [hunk 16]
```diff
@@ -3,78 +3,68 @@ import { NextSeo } from "next-seo";
...
     return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase();
   });
 };
 
-const Masonry = styled("section", {
-  width: "100%",
-  display: "grid",
-  gridTemplateColumns: "repeat( auto-fit, minmax(250px, 1fr) )",
-  gridGap: "$100",
+const HeadDiv = styled("div", {
```
Comment (atomictangerine) [inline pages/resources/[category]/index.js:20]: 😻 Really didn't like the Masonry name. Very confusing.

File: pages/resources/[category]/index.js [hunk 17]
```diff
@@ -88,11 +78,31 @@ export const getStaticProps = async ({ params }) => {
 
   const navigation = await getNavigation();
 
+  // populate props
+  let description, type, size;
+  type = params.category;
```
Comment (atomictangerine) [inline pages/resources/[category]/index.js]: could we move the definition of type to this line? no need to have it twice.
`let type = params.category'`

File: pages/resources/[category]/index.js [hunk 18]
```diff
@@ -88,11 +78,31 @@ export const getStaticProps = async ({ params }) => {
 
   const navigation = await getNavigation();
 
+  // populate props
+  let description, type, size;
```
Comment (atomictangerine) [inline pages/resources/[category]/index.js]: can we initiate these to an empty string?

File: pages/resources/[category]/index.js [hunk 19]
```diff
@@ -88,11 +78,31 @@ export const getStaticProps = async ({ params }) => {
...
+  type = params.category;
+  if (type === "tutorials") {
+    description =
+      "Watch or read through our tutorials to understand key techniques and concepts.";
+    size = "full";
```
Comment (atomictangerine) [inline pages/resources/[category]/index.js]: where we would import and use that constant from thumbnails. `size = THUMBNAIL_WIDE`

File: pages/resources/index.js [hunk 20]
```diff
@@ -156,12 +155,75 @@ export const getStaticProps = async ({ params }) => {
...
+      posts = [...collections[i].docs]
+        .sort(function (a, b) {
+          try {
+            return a.data.guideRank - b.data.guideRank;
+          } catch (TypeError) {
+            return a.data.title.localeCompare(b.data.title);
+          }
+        })
+        .slice(0, 4);
+      size = "mini";
```
Comment (atomictangerine) [inline pages/resources/index.js]: another great instance where we could use `THUMBNAIL`

File: pages/resources/index.js [hunk 21]
```diff
@@ -156,12 +155,75 @@ export const getStaticProps = async ({ params }) => {
...
     },
   };
 };
+
+async function getProps(collections, recents) {
+  // create a wrapper which contains all necessary data for the page
+  // includes page content based on section (content) and headings to pass to the TOC
+  // we have to hard-code What's New because it's too unique
+  let wrapper = { content: {}, headings: [] };
+  let content = { categories: [] };
```
Comment (atomictangerine) [inline pages/resources/index.js]: I think you can use a const here, since you're not actually changing the content object but instead pushing to the categories array inside of it

File: pages/resources/index.js [hunk 22]
```diff
@@ -156,12 +155,75 @@ export const getStaticProps = async ({ params }) => {
...
+  const descriptions = {
+    Guides:
+      "Explore the processes and tools we use in our step-by-step written guides.",
+    Tutorials:
+      "Watch or read through our tutorials to understand key techniques and concepts.",
+    Workshops:
+      "Sharpen your design and development skills with our in-depth recorded workshops.",
+  };
+
+  for (let i = 0; i < collections.length; i++) {
```
Comment (atomictangerine) [inline pages/resources/index.js]: Is there a particular reason we're doing a for loop instead of mapping over the collections array?

File: pages/resources/index.js [hunk 23]
```diff
@@ -156,12 +155,75 @@ export const getStaticProps = async ({ params }) => {
...
+  let content = { categories: [] };
+  let [name, posts, description, size, type, id] = [
+    "What's New?",
+    recents,
+    null,
+    "full",
+    "New",
+    "What's%20New?",
+  ];
+  let category = { name, posts, description, size, type, id };
```
Comment (atomictangerine) [inline pages/resources/index.js]: It looks like you're using category only inside the loop. I would move these declarations inside the loop since that's you don't need them to persist outside of that.

File: pages/resources/index.js [hunk 24]
```diff
@@ -156,12 +155,75 @@ export const getStaticProps = async ({ params }) => {
...
   return {
     props: {
-      collections,
       navigation,
+      wrapper,
     },
   };
 };
+
+async function getProps(collections, recents) {
```
Comment (atomictangerine) [inline pages/resources/index.js:176]: I think this function could be refactored a little bit for clarity. It's not very human-friendly to read. (Looks more like a python function than a JS one ;) )

File: pages/support/index.js [hunk 25]
```diff
@@ -0,0 +1,125 @@
...
+const Divider = styled("hr", {
+  gridColumnEnd: "span 2",
+  margin: "$050 0 $050",
+  padding: 0,
+  border: 0,
+  height: "1px",
+  backgroundColor: "$subtle",
+});
+
+const ChevronForLink = styled(ChevronRight, {
```
Comment (atomictangerine) [inline pages/support/index.js]: This is the third time I see this `ChevronForLink` being created. Always the same. Could we pull it out and reuse it?

**Review by atomictangerine — CHANGES_REQUESTED** [review]

File: components/Thumbnail.js (hunk 14 above)
Comment (clairehhelms) [reply components/Thumbnail.js]: this is cool!!

File: pages/resources/[category]/index.js (hunk 15 above)
Comment (clairehhelms) [reply pages/resources/[category]/index.js]: yes!

File: pages/resources/index.js (hunk 21 above)
Comment (clairehhelms) [reply pages/resources/index.js]: good point!

File: pages/support/index.js (hunk 25 above)
Comment (clairehhelms) [reply pages/support/index.js]: lol I just took it out! I didn't write this page, but it wasn't being used 🤷

**Review by atomictangerine — APPROVED** [review]
LGTM
